source/common/router/config_impl.h

```diff
@@ -15,7 +15,9 @@
 #include "envoy/runtime/runtime.h"
 #include "envoy/upstream/cluster_manager.h"
 
+#include "common/http/access_log/access_log_formatter.h"
```

Comment:
Is this header needed? What is it needed for? If not needed please remember to remove from BUILD file also.